test/abagnale.js

```diff
@@ -4,6 +4,38 @@ import fs from 'fs';
 import glob from 'glob';
 import path from 'path';
 
+describe('Bad refract input data', () => {
+  it('should not crash', () => {
```

Comment:
One small idea: it may be helpful to wrap in a context that describes the nature of the bad input. For instance:

```
context('when a class is null`, () => {})
```